### PR TITLE
Sign-In-Updates: Updated styling of login pages, fixed create team pop-over styling

### PR DESCRIPTION
**Pull request by ttseng**
## Links
* Remix link: https://sign-in-updates.glitch.me/
* Issue-tracker link: 
https://app.clubhouse.io/glitch/story/4511/update-sign-in-styling
https://app.clubhouse.io/glitch/story/4280/create-team-pop-width-incorrect

## GIF/Screenshots:
image
* Edited oauth button ordering to put Facebook last
* made button size of login items consistent
* increased margin around terms and conditions

image
* fixed pop-over title text size

image
Fixed teams pop-over width

## How To Test:
* https://sign-in-updates.glitch.me/login - check that things look consistent with the changes described above
* https://sign-in-updates.glitch.me/ - log in and check out the create-teams-pop

### CODE REVIEW

**Comment by ttseng** [comment]
I added more design to the `/login` page.  This is now ready for review!

login

@oliviacpu I think we should rename `'/login` to `/signin` but I'm afraid to break things - do you know the best place to do this?

The SVG files that are being used were uploaded directly to prod:

Animated SVG background: https://cdn.glitch.com/0aa2fffe-82eb-4b72-a5e9-444d4b7ce805%2Fsign-in-pattern.svg?v=1574364848746

Static SVG background (for when `prefers-reduced-motion` is on): https://cdn.glitch.com/0aa2fffe-82eb-4b72-a5e9-444d4b7ce805%2Fsign-in-pattern-static.svg?v=1574364997266

File: src/components/layout/sign-in-layout.styl [hunk 1]
```diff
@@ -1,15 +1,31 @@
...
+  background-size: 400px
+  @media(prefers-reduced-motion: reduce){
+    // static background SVG that doesn't contain animation
+    background-image: url('https://cdn.glitch.com/0aa2fffe-82eb-4b72-a5e9-444d4b7ce805%2Fsign-in-pattern-static.svg?v=1574364997266')
+  }
+
 .overlay
+  position: initial
   max-width: 320px
+  transform: inherit
```
Comment (ehmorris) [inline src/components/layout/sign-in-layout.styl]: What is this property doing?

File: src/components/layout/sign-in-layout.styl [hunk 2]
```diff
@@ -25,6 +41,20 @@
...
   color: tertiary
   padding: 0.5em 0
+  margin: 1em 0 0
 
 .helpText
-  margin: 0 0 0.5em 0
+  margin: 0 0 1em 0
+
+.extra
+    background: primary-background
```
Comment (ehmorris) [inline src/components/layout/sign-in-layout.styl]: Looks like these properties are indented more than the others

File: src/components/overlays/overlays.styl [hunk 3]
```diff
@@ -33,7 +33,7 @@ pop-shadow = 0px 2px 5px 0px rgba(0,0,0,0.27), 0px 1px 1px 0px rgba(0,0,0,0.15)
   background-color: secondary-background
 
 .title
-  font-size: inherit
+  font-size: initial
```
Comment (ehmorris) [inline src/components/overlays/overlays.styl]: I think setting this to a font size variable might be better than using `initial`

File: src/components/layout/sign-in-layout.styl (hunk 1 above)
Comment (ttseng) [reply src/components/layout/sign-in-layout.styl]: there are some defaults transforms in the `overlay` component (/src/components/overlay/overlay.styl) that aren't compatible with the flex layout that I'm using here, so I'm overriding it.

Comment (ehmorris) [reply src/components/layout/sign-in-layout.styl]: Ahh I see what you mean, because you had to override the fixed position as well.

**Pull request merged**